sys/arduino/base.cpp

```diff
@@ -106,3 +107,50 @@ int analogRead(int arduino_pin)
     return adc_value;
 }
 #endif
+
+#if MODULE_PERIPH_PWM
+#define PWM_MODE            PWM_LEFT
+#define PWM_FREQU           (1000U)
+#define PWM_STEPS           (256U)
```

Comment:
You are right, I also read that frequency values depend on the boards in Arduino. This could be moved elsewhere: default defines in the Arduino.hpp and values overridden in boards to match the ones in official Arduino.

Reply:
sounds great

Reply:
Until now, there is no `Arduino.hpp`. It will be introduced for compatibility reasons with PR #10592 which is waiting for merge. But `Arduino.hpp` will be just a wrapper and all definitions should got to `arduino.hpp`.

Reply:
I moved the defines in arduino.hpp (don't worry @gschorcht, there was a typo in my previous comment). Only frequency can be overridden. The other values seems to be fixed in Arduino.